### DISCUSSION TITLE
NextclouPi - The PHP memory limit is below the recommended value of 512MB

### DISCUSSION THREAD

**Discussion by Vendo232** [tteck/Proxmox#1344] [Q&A] [answered]
Just discovered this repo and tried right away the NextcloudPi which was super easy to install and the image already includes SMB support not like the Docker one.

Inside the NextcloudPi I`m getting this msg : "The PHP memory limit is below the recommended value of 512MB"

any suggestion to fix it?

**Comment by Vendo232** [comment]
changed the following

in /etc/php/8.1/CLI and FPM

upload_max_filesize=1G
post_max_size=1G
memory_limit=512M

restart

**Reply by tteck** [reply, marked as answer]
in `/etc/php/8.1/cli/php.ini` the default is `memory_limit = -1` which means unlimited
However, it is generally a good practice to set a reasonable memory limit for your scripts to avoid issues with excessive memory usage.

in `/etc/php/8.1/fpm/php.ini` the default is `memory_limit = 128M` which needs to be increased to `memory_limit = 512M`

This command sets both to `memory_limit = 512M`
```
sed -i 's/memory_limit = .*/memory_limit = 512M/' /etc/php/8.1/fpm/php.ini /etc/php/8.1/cli/php.ini
```
Will be adding to the install script
